css/properties.json

```diff
@@ -2748,6 +2820,30 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/border-image-width"
   },
+  "border-inline": {
+    "syntax": "<'border-width'> || <'border-style'> || <'color'>",
```

Comment:
according to the spec, should be `<'border-top-width'> || <'border-top-style'> || <'color'>` ?